jekyll/_cci2/hello-world-windows.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: classic-docs
+title: "Hello World On Windows"
+short-title: "Hello World On Windows"
+description: "First Windows project on CircleCI 2.0"
+categories: [getting-started]
+order: 4
+---
+
+This document describes how to get started with continuous integration on **Windows build environments** on CircleCI. If this is your first time setting up CircleCI, we recommended checking out the [getting started guide.]({{ site.baseurl}}/2.0/getting-started/) Please note, Windows access is currently in a **preview phase** and is **not** production ready.
+
+# Prerequisites
+
+To follow along with this document you will need:
+
+* An [account](https://circleci.com/signup/) on CircleCI
+* A [performance plan](https://circleci.com/pricing/usage/) subscription
+* Pipelines must be [enabled]({{site.baseurl}}/2.0/build-processing/) (to enable Orbs and a 2.1 config).
+* To request access to Windows Preview using [this form](https://docs.google.com/forms/d/e/1FAIpQLSfspug2MP0eTK8eRC1_FpiDQzNHkk8a36fflN_za29CwGzGoQ/viewform).
+
+
+# Overview of the Windows executor
+
+The Windows build environment (or `executor`) gives users the tools to build Windows projects, such as a Universal Windows Platform (UWP) application, a .NET executable, or Windows-specific (like the .NET framework) The Windows executor uses the Server Core version of Windows Server 2019 DataCenter Edition.
+
+A windows executor will have 4 vCPUs and 15GB of ram. Currently, there are no I/O optimizations.
+
+On the Windows executor, _Git_, the package manager _Chocolatey_, and _7zip_ will be installed. Powersell is the default shell (Bash and Command are available to be manually selected). Other dependencies may be installed later on.
```

Comment:
`Powersell is the default shell (Bash and Command are available to be manually selected).` - typo, should be Powershell